modin/test/storage_formats/pandas/test_internals.py

```diff
@@ -757,3 +757,47 @@ def test_repartitioning(set_num_partitions):
     assert res.column_widths == [4]
     df_equals(res._partitions[0, 0].to_pandas(), pandas_df)
     df_equals(res.to_pandas(), pandas_df)
+
+
+@pytest.mark.parametrize("col_name", ["numeric_col", "non_numeric_col"])
+@pytest.mark.parametrize("ascending", [True, False])
+@pytest.mark.parametrize("num_pivots", [3, 2, 1])
+@pytest.mark.parametrize("all_pivots_are_unique", [True, False])
+def test_split_partitions_kernel(
```

Comment:
A fairly complex test, a couple of comments right in it will not hurt to make it easier to understand.